rasa/core/agent.py

```diff
@@ -504,10 +504,7 @@ async def handle_text(
 
     def _set_fingerprint(self, fingerprint: Optional[Text] = None) -> None:
 
-        if fingerprint:
-            self.fingerprint = fingerprint
-        else:
-            self.fingerprint = uuid.uuid4().hex
+        self.fingerprint = fingerprint or uuid.uuid4().hex
```

Comment:
Function `Agent._set_fingerprint` refactored with the following changes:

- Simplify if expression by using or ([or-if-exp-identity](https://docs.sourcery.ai/refactorings/or-if-exp-identity/))
- Replace if statement with if expression ([assign-if-exp](https://docs.sourcery.ai/refactorings/assign-if-exp/))